### DISCUSSION TITLE
Why i got 405 of not found api?

### DISCUSSION THREAD

**Discussion by seepine** [fastify/fastify#5897] [Q&A]
I route `POST:  /info`, and `curl /info` got 405, `curl /info2` got 405 also.

**Comment by Eomm** [comment]
Could you add a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)?

Without it, we are unable to help you.

**Reply by seepine** [reply]
Alright, if this isn't designed this way, then I'll recreate a minimal project to give it a try.

**Comment by seepine** [comment]
Sorry, it intercepted by a hook.
